resources/puppetlabs/lein-ezbake/template/global/ext/cli/reload.erb

```diff
@@ -5,10 +5,12 @@ restartfile=<%= EZBake::Config[:restart_file] %>
 timeout=<%= EZBake::Config[:start_timeout] %>
 
 if [ -r "$restartfile" ];  then
-    kill -HUP $(pgrep -f <%= EZBake::Config[:uberjar_name] %>)
+    pid=$(pgrep -f <%= EZBake::Config[:uberjar_name] %>)
+    kill -HUP $pid
     cur=$(head -n 1 "$restartfile")
     initial=$cur
     while [ "$cur" == "$initial" ] ;do
+        kill -0 $pid >/dev/null 2>&1
```

Comment:
I went ahead and added the trace message in f7b5fe8.  Also removed the `set -e` from the entire script since exit codes are generally handled anyway so it didn't seem necessary anymore.